openqa-trigger/Makefile

```diff
@@ -0,0 +1,5 @@
+build:
```

Comment:
This could easily be a `docker-compose.yml` where we can add the rest of the services in this repo right?

Reply:
Done. Better with the `docker-compose.yml` file, yes.